src/Backoffice/Flights/Domain/Actions/DeleteFlightAction.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Flights\Domain\Actions;
+
+use Lightit\Backoffice\Flights\Domain\Models\Flight;
+
+class DeleteFlightAction
+{
+    public function execute(Flight $flight): bool|null
+    {
+        return $flight->delete();
+    }
```

Comment:
you arent using the return in the controller, you can just put `: void` and not return anything